### PR TITLE
read unit and description from netcdf

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

File: services/src/datasets/external/netcdfcf/mod.rs [hunk 1]
```diff
@@ -1205,7 +1239,7 @@ mod tests {
                     .to_string(),
                 }),
                 name: "Test dataset metric: Random metric 1 > entity01".into(),
-                description: "".into(),
+                description: "CFake description of test dataset with metric.".into(),
```
Comment (jdroenner) [inline services/src/datasets/external/netcdfcf/mod.rs:1242]: typo

File: services/src/datasets/external/netcdfcf/mod.rs [hunk 2]
```diff
@@ -1225,7 +1259,7 @@ mod tests {
                     .to_string(),
                 }),
                 name: "Test dataset metric: Random metric 1 > entity02".into(),
-                description: "".into(),
+                description: "CFake description of test dataset with metric.".into(),
```
Comment (jdroenner) [inline services/src/datasets/external/netcdfcf/mod.rs:1262]: typo

File: services/src/datasets/external/netcdfcf/mod.rs [hunk 3]
```diff
@@ -1245,7 +1279,7 @@ mod tests {
                     .to_string(),
                 }),
                 name: "Test dataset metric: Random metric 1 > entity03".into(),
-                description: "".into(),
+                description: "CFake description of test dataset with metric.".into(),
```
Comment (jdroenner) [inline services/src/datasets/external/netcdfcf/mod.rs:1282]: typo

File: services/src/datasets/external/netcdfcf/mod.rs [hunk 4]
```diff
@@ -1265,7 +1299,7 @@ mod tests {
                     .to_string(),
                 }),
                 name: "Test dataset metric: Random metric 2 > entity01".into(),
-                description: "".into(),
+                description: "CFake description of test dataset with metric.".into(),
```
Comment (jdroenner) [inline services/src/datasets/external/netcdfcf/mod.rs:1302]: typo

**Review by jdroenner — CHANGES_REQUESTED** [review]

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Comment by jdroenner** [comment]
bord merge

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**